src/MarginTrading.Backend/Controllers/OvernightSwapController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using MarginTrading.Backend.Core;
+using MarginTrading.Backend.Core.Repositories;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace MarginTrading.Backend.Controllers
+{
+	[Authorize]
+	[Route("api/overnightswap")]
+	public class OvernightSwapController : Controller
+	{
+		private readonly IOvernightSwapHistoryRepository _overnightSwapHistoryRepository;
+		
+		public OvernightSwapController(IOvernightSwapHistoryRepository overnightSwapHistoryRepository)
+		{
+			_overnightSwapHistoryRepository = overnightSwapHistoryRepository;
+		}
+		
+		[Route("overnight.swaps.state")]
```

Comment:
Strange route api/overnightswap/overnight.swaps.state. 
And it is not state, but history.. 
May be change to [Route("history")] ?

Reply:
Fixed